pages/index.tsx

```diff
@@ -227,12 +227,20 @@ function Display({
 
 const Status = (props: SubgraphIndexingStatus) => {
   const chain = props.chains[0]
+  const [state, copyToClipboard] = useCopyToClipboard()
```

Comment:
i understand `useCopyToClipboard` example uses the name `state` but i find it a little confusing.

How about naming it `clipboard` to be super straightforward?